docs/pages/components/side-navigation.md

```diff
@@ -198,31 +201,32 @@ On expand, the `is-expanded` class should be propagated also to `__content` elem
 ## Side navigation with 2 levels - with icons and group headers, cozy mode
 
 {% capture default %}
-<nav class="fd-side-nav">
-    <div class="fd-side-nav__main-navigation">
-        <ul class="fd-nested-list">
-            <li class="fd-nested-list__group-header">
-                Group Header 1
-            </li>
+<div class="fd-side-nav">
+    <a class="fd-side-nav__skip-link" href="#content">Skip navigation</a>
+    <div class="fd-side-nav__group-header" id="EX400H1">
+        Group Header 1
+    </div>
+    <nav class="fd-side-nav__main-navigation">
+        <ul class="fd-nested-list" aria-labelledby="EX400H1">
             <li class="fd-nested-list__item">
                 <a class="fd-nested-list__link" href="#">
-                    <span class="fd-nested-list__icon sap-icon--home"></span>
+                    <span aria-hidden="true" class="fd-nested-list__icon sap-icon--home"></span>
                     <span class="fd-nested-list__title">Level 1 Item</span>
                 </a>
             </li>
             <li class="fd-nested-list__item">
                 <a class="fd-nested-list__link" href="#">
-                    <span class="fd-nested-list__icon sap-icon--calendar"></span>
+                    <span aria-hidden="true" class="fd-nested-list__icon sap-icon--calendar"></span>
                     <span class="fd-nested-list__title">Level 1 Item</span>
                 </a>
             </li>
             <li class="fd-nested-list__item">
-                 <div class="fd-nested-list__content is-selected has-child" tabindex="0">
-                    <a class="fd-nested-list__link" href="#" tabindex="-1">
-                        <span class="fd-nested-list__icon sap-icon--employee"></span>
+                <div class="fd-nested-list__content is-selected has-child">
+                    <span class="fd-nested-list__link">
+                        <span aria-hidden="true" class="fd-nested-list__icon sap-icon--employee"></span>
                         <span class="fd-nested-list__title">Level 1 Item</span>
-                    </a>
-                    <a class="fd-nested-list__expand-icon" href="#" aria-controls="EX400L2" tabindex="-1" aria-haspopup="true"></a>
+                    </span>
+                    <button class="fd-nested-list__expand-icon" aria-controls="EX100L2" aria-haspopup="true" aria-expanded="false"></button>
```

Comment:
In the past, we've used the pattern `<button class='fd-button fd-nested-list__button'`` so that the implementation libraries can reuse their existing buttons without creating one-off functionality